patterns/text-only-blog-home-template.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Title: Text-only blog, home
+ * Slug: twentytwentyfive/text-only-blog-home
+ * Template Types: front-page, home
+ * Viewport width: 1400
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:template-part {"slug":"header","tagName":"header","area":"header"} /-->
+
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--60)">
+	<!-- wp:heading {"level":1,"align":"wide","fontSize":"large"} -->
+	<h1 class="wp-block-heading alignwide has-large-font-size"><?php esc_html_e( 'Blog', 'twentytwentyfive' ); ?></h1>
```

Comment:
I don't see this in the design, but I assume it's to have a proper hierarchy? 

What I noticed is that it is the same size as the title of the blog posts.